src/main/resources/db/migration/V202409111400__add_owner_id_column_to_pets.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE pets
+ADD COLUMN owner_id UUID
```

Comment:
Better and safer to add `if not exists`
i.e.
ALTER TABLE someTable ADD COLUMN if not exists someColumn TYPE

Reply:
thats a comment for the future? or should i delete this migration and make new one?

Reply:
No need to delete- this file is not yet in master - right?
So you can change it
Adding `if not exists` might affect your locally  - if this is already in the flyway history table - but this can be handled localy